Monika After Story/game/script-topics.rpy

```diff
@@ -535,6 +521,43 @@ label mas_topic_rerandom:
     return
 
 init python in mas_bookmarks_derand:
+    import store
+
+    #Rule constants
+    WHITELIST = "whitelist_bookmark"
+    BLACKLIST = "blacklist_bookmark"
```

Comment:
Since the intent of these is to be used in rules, what happens if a topic has both of these? I think this would be better as a single key, and use the value to determine wl/bl.